### ISSUE TITLE
[Whitelist] Titan Quest (4540)

### ISSUE THREAD

**Issue by liberodark** [ValveSoftware/Proton#949]
Distro: Manjaro XFCE 17.1.12
GPU: GTX 1060, Nvidia proprietary driver 396.54
Resolution: 1920x1080
RAM: 8GB
Proton 3.7-4 Beta

All is fine.

**Comment by MaxXor** [comment]
Problems here on Arch, characters are invisible. I can just see the armour and weapons. :smile: 
https://github.com/ValveSoftware/Proton/issues/567 this does the trick.

**Comment by liberodark** [comment]
You are wrong #567 speak to the aniversary edition for me not work / no run.
im speak to the original version is not same.
4540 for me work very great :D no problem
but 475150 not work for me !
manjaro is based on arch
